packages/manager/apps/telecom/src/app/telecom/telephony/alias/special/repayments/repayments.controller.js

```diff
@@ -31,13 +39,16 @@ export default class TelecomTelephonyAliasSpecialRepaymentsCtrl {
 
     this.loading = true;
 
-    return this.OvhApiTelephony.Service()
-      .RepaymentConsumption()
-      .Aapi()
-      .repayment({
+    return this.OvhHttp.get('/telephony/{billingAccount}/repayment', {
```

Comment:
Both `OvhHttp` and `OvhApi*` services are deprecated.

We tend to use `$http`. Is it possible to use it here?